zerver/views/__init__.py

```diff
@@ -1881,6 +1881,12 @@ def json_fetch_api_key(request, user_profile, password=REQ(default='')):
         return json_error("Your username or password is incorrect.")
     return json_success({"api_key": user_profile.api_key})
 
+@csrf_exempt
+def api_fetch_google_client_id(request):
+    if not settings.GOOGLE_CLIENT_ID:
+        return json_error("GOOGLE_CLIENT_ID is not configured", status=400)
```

Comment:
@leah: if I have an API endpoint for clients to retrieve some sort of configuration information from my server, but some servers may not have such configuration information defined, is the right answer to return 400, 404, or 200 but with a body indicating not found?